MSGraphCoreSDK/MSGraphCoreSDK/GraphContent/BatchContent/MSBatchRequestContent.m

```diff
@@ -0,0 +1,168 @@
+//
+// Copyright (c) Microsoft Corporation. All Rights Reserved. Licensed under the MIT License. See License in the project root for license information.
+//
+
+#import "MSBatchRequestContent.h"
+#import "MSBatchRequestStep.h"
+#import "MSConstants.h"
+#import "MSErrorCodes.h"
+
+@interface MSBatchRequestContent()
+
+@property (strong, nonatomic) NSMutableArray<MSBatchRequestStep *> * batchRequestStepsArray;
+
+@end
+
+@implementation MSBatchRequestContent
+{
+    NSInteger maxNumberOfRequests;
+}
+
+- (instancetype)initWithRequests:(NSArray<MSBatchRequestStep *> *)batchRequestStepsArray error:(NSError *__autoreleasing *)error
+{
+    self = [super init];
+    if(self)
+    {
+        maxNumberOfRequests = 20;
```

Comment:
can you put a comment that 20 is a enforcement by graph

Reply:
done.